default.nix

```diff
@@ -0,0 +1,28 @@
+{ # Fetch the latest haskell.nix and import its default.nix
+haskellNix ? (import (import ./nix/sources.nix)."haskell.nix" { })
+
+# haskell.nix provides access to the nixpkgs pins which are used by our CI,
+# hence you will be more likely to get cache hits when using these.
+# But you can also just use your own, e.g. '<nixpkgs>'.
+, nixpkgsSrc ? haskellNix.sources.nixpkgs-2009
+
+  # haskell.nix provides some arguments to be passed to nixpkgs, including some
+  # patches and also the haskell.nix functionality itself as an overlay.
+, nixpkgsArgs ? haskellNix.nixpkgsArgs, compiler-nix-name ? "ghc884" }:
+let
+  pkgs' = import <nixpkgs> { };
+  # Define custom nixpkgs overlay
+  darwinOverlays = if pkgs'.stdenv.isDarwin then
+    [ (self: super: { sdl2 = super.haskellPackages.sdl2; }) ]
```

Comment:
This still doesn't look right. This is saying that `pkgs.sdl2` - the `libsdl2` C library - is the Haskell package `sdl2`. These aren't the same thing. Are you sure you need this?